docs/git-guide.md

```diff
@@ -1357,6 +1357,47 @@ HEAD is now at 99aa2bf Add provision.py fails issue in common erros
 Current branch review-1913 is up to date.
 ```
 
+### Fetch a pull request without rebasing
+
+`tools/fetch-pull-request` is a similar to `tools/fetch-rebase-pull-request`, but
+it does not rebase the pull request with any changes from upstream/master,
+thereby getting exactly the same repository state as the commit author had.
+
+First, make sure you are working in branch `master`. Then run the script with
```

Comment:
Why do you have to make sure you are working in branch `master`? This should work on any branch.